### PR TITLE
Merch delete item tg

### PR DESCRIPTION
**Pull request by tyladevon**
Merch can delete item that has never been ordered.

### CODE REVIEW

File: app/controllers/merchant/items_controller.rb [hunk 1]
```diff
@@ -15,4 +15,13 @@ def update
     end
     redirect_to '/merchant/items'
   end
+
+  def destroy
+    item = Item.find(params[:id])
+    if item.no_orders?
```
Comment (hillstew) [inline app/controllers/merchant/items_controller.rb]: Is this check different than the one in the view? If we check it once in the view, we wouldn't have to recheck in our destroy method?

Comment (rhantak) [reply app/controllers/merchant/items_controller.rb]: Plus one to this

Comment (tyladevon) [reply app/controllers/merchant/items_controller.rb]: Okay. Thanks for the check! I will fix it!

File: app/views/merchant/items/index.html.erb [hunk 2]
```diff
@@ -5,6 +5,9 @@
     <h3><%= item.name %></h3>
     <p>Price: <%= number_to_currency(item.price) %></p>
     <p>Inventory: <%= item.inventory %></p>
+    <% if item.no_orders? %>
```
Comment (rhantak) [inline app/views/merchant/items/index.html.erb]: Love that you used a method here; much better than item.orders.empty? living in the view

**Pull request closed without merging**